### ISSUE TITLE
Add label to splash screen for accessibility compliance

### ISSUE THREAD

**Issue by kickbk** [expo/expo#13663]
### Summary

I'm getting a warning in Google Play store for the splash screen with [This item may not have a label readable by screen readers.](https://support.google.com/accessibility/android/answer/7158690)
I would dismiss this as low priority, but recently a lot of people get sued for non ADA compliance, including a client of mine, and it's expensive.
Unless there's already a way to add a label to the splash screen image (if so please advise), I would recommend adding a new "label" option via `app.json` under `splash`. 

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

Android

### SDK Version (managed workflow only)

42

### Environment

```
Expo CLI 4.7.3 environment info:
    System:
      OS: macOS 11.4
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 14.17.3 - ~/.nvm/versions/node/v14.17.3/bin/node
      Yarn: 1.22.10 - /usr/local/bin/yarn
      npm: 7.19.1 - ~/.nvm/versions/node/v14.17.3/bin/npm
      Watchman: 2021.06.07.00 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.10.1 - /Users/benkass/.rvm/gems/ruby-2.7.1/bin/pod
    SDKs:
      iOS SDK:
        Platforms: iOS 14.5, DriverKit 20.4, macOS 11.3, tvOS 14.5, watchOS 7.4
      Android SDK:
        API Levels: 29
        Build Tools: 23.0.2, 29.0.2, 29.0.3, 30.0.0, 31.0.0
        System Images: android-30 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 4.2 AI-202.7660.26.42.7486908
      Xcode: 12.5.1/12E507 - /usr/bin/xcodebuild
    npmPackages:
      expo: ~42.0.0 => 42.0.1 
      react: 16.13.1 => 16.13.1 
      react-dom: 16.13.1 => 16.13.1 
      react-native: https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz => 0.63.2 
    npmGlobalPackages:
      expo-cli: 4.7.3
    Expo Workflow: managed
```

### Reproducible demo or steps to reproduce from a blank project

Any managed expo project will do

**Comment by rodrigohgpontes** [comment]
Please reopen this issue. It is still a problem on expo 45 (when you use an image for the splash page on `app.json` expo config.

**Comment by brentvatne** [comment]
i'd recommend investigating how to build this using config plugins: https://docs.expo.dev/guides/config-plugins/

the implementation for the splash screen config plugin lives here: https://github.com/expo/expo-cli/tree/main/packages/prebuild-config/src/plugins/unversioned/expo-splash-screen

if you'd like to send a pr to add support for it this would be welcome!

**Comment by moxen-dev** [comment]
Same issue here.

@brentvatne the problem is related when not using this library.

I guess the app config file should have a key to add this info so we can move on without the necessity to add this library.

**Comment by patrickalima98** [comment]
Hello @moxen-dev, I'm in Expo 48 and I have the same problem. Did you manage to solve?

**Comment by moxen-dev** [comment]
No but Google accepted the app anyway. I guess the solution is to use this expo external library.
